### DISCUSSION TITLE
[Feature] put together faces that belong to the same person

### DISCUSSION THREAD

**Discussion by juanpeich** [immich-app/immich#11022] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

immich recognizes faces in photos, and although it can be edited on each photo. Could the option be put in the People section to unify found faces that belong to the same person? Thanks in advance JuamPe

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The menu on a person lets you merge them.

![image](https://github.com/immich-app/immich/assets/11602424/576dfc17-0d5c-4f08-8709-bca614e78109)

**Comment by Echsecutor** [comment]
Where can I find this feature in the Android App?
